prioritizer/utils/reading_data.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+import json
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+ACTION_DATA_PATH = BASE_DIR / "CAP_data/long_actions.json"
+CITY_DATA_PATH = BASE_DIR / "CAP_data/city_data.json"
+OUTPUT_FILE = BASE_DIR / "new_output.json"
+
+
+def read_city_inventory():
+    """
+    Reads city inventory data from a JSON file.
+
+    Returns:
+        dict: The city inventory data.
+    """
+    city_data_path = CITY_DATA_PATH
+    with city_data_path.open("r", encoding="utf-8") as f:
+        city_data = json.load(f)
+    return city_data[0]  # Adjust as needed for multiple cities
```

Comment:
Maybe its easier to pass in a city name here?
e.g. def read_city_inventory(cityName: str)
and then retrieve the city from the city_data.get(CityName) or similar?

Otherwise when we have lots of cities and we want to see do this for a specific city, we would need to check first in the data file the index of the city.

in your if __name__ == "__main__": you could then implement argparse and pass in the city name as an argument which would make this script easier to use without changing the actual code (adapting the index)